### PR TITLE
Better event management

### PR DESCRIPTION
**Pull request by jourdain**

### CODE REVIEW

**Comment by jourdain** [comment]
FYI @ChristosT

File: trame_rca/protocol.py [hunk 1]
```diff
@@ -68,6 +68,8 @@ def push_content(self, area_name, metadata, content):
     def on_interaction(self, area_name, origin, event):
         adapter = self._area_adapters.get(area_name, None)
         if adapter:
-            adapter.on_interaction(origin, event)
+            # FIXME
+            print("on_interaction", event)
+            # adapter.on_interaction(origin, event)
```
Comment (jourdain) [inline trame_rca/protocol.py:73]: @ChristosT this is where you need to connect the server side... Probably need to uncomment the code and fix the async part to understand the events you get.

**Comment by ChristosT** [comment]
The example can be tested with a wheel from https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10545

**Review by jourdain — COMMENTED** [review]
LGTM

**Pull request merged**